queries/latex/highlights.scm

```diff
@@ -1,3 +1,9 @@
+; Formatting
+(text_mode
+  content:
+    (curly_group
+      (_) @none @spell))
```

Comment:
Do we really need to move this to top?

Reply:
It just needs to be before the `@markup.math` highlights, I'll just put it right above them. I think grouping them like that will make more sense. I will break up the commits as well like you say :+1: